src/GraphQL/Execution/DocumentExecuter.cs

```diff
@@ -148,9 +148,12 @@ private async Task<ExecutionResult> CoreExecuteAsync(ExecutionOptions options)
 
                 if (!validationResult.IsValid)
```

Comment:
Can it be `if (!validationResult.IsValid || context.Errors.Count > 0)` and the lines 161-168 removed?

Reply:
Was trying to reduce allocations, but i can still simplify it some

Reply:
Simplified; harder to understand now but less code